### PR TITLE
Add support for volumetric flow rate units.

### PR DESCRIPTION
**Pull request by splatch**
First units is SI `m³/s`. Additional unit are `m³/h` and `L/s` for consumer flow meters. Solves #6549.

### CODE REVIEW

File: bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/unit/SmartHomeUnits.java [hunk 1]
```diff
@@ -164,6 +171,9 @@ public String getName() {
         SimpleUnitFormat.getInstance().label(MILLIBAR, "mbar");
         SimpleUnitFormat.getInstance().label(KNOT, KNOT.getSymbol());
         SimpleUnitFormat.getInstance().label(DOBSON_UNIT, "DU");
+        SimpleUnitFormat.getInstance().label(CUBICMETRE_PER_SECOND, "m³/s");
+        SimpleUnitFormat.getInstance().label(LITRE_PER_MINUTE, "L/min");
```
Comment (J-N-K) [inline bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/unit/SmartHomeUnits.java]: The unit symbol for Units.LITRE is „l“, not „L“. We should use the same here.

Comment (splatch) [reply bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/unit/SmartHomeUnits.java]: I understand your point, however `L` is also widely accepted and I am not sure if its not more popular in some circles due to readability. It is easier to read it in this for that l (`l`) which is quite similar to I (`I`). I can change to `l/min`, however at the same time I would ask why not going with `L` in place you mentioned?

Comment (J-N-K) [reply bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/unit/SmartHomeUnits.java]: Wikipedia says that both are allowed. „l“ was the original symbol, in 1979 „L“
was allowed as alternative symbol. 

The upper-case L is mostly used in US/Canada while the lower-case l is used in Europe. I‘m not sure if we can allow both, what I would suggest.

Comment (htreu) [reply bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/unit/SmartHomeUnits.java]: UoM currently does not support multiple formats for the same unit. I would prefer the `l` variant.

**Comment by splatch** [comment]
> Could you add some tests?

@J-N-K I'm not sure if test for presence of constants in any class brings any value. What is your take on this?

**Comment by J-N-K** [comment]
Presence of constants not. But the tests could/should cover the parsing of a string with that unit to the equivalent QuantityType and the correct conversion to or from that unit to others

**Comment by splatch** [comment]
I've added tests for parsing these units and conversion between them. Please review.

**Pull request merged**